server/athenian/api/align/queries/teams.py

```diff
@@ -23,19 +23,19 @@ async def resolve_teams(obj: Any, info: GraphQLResolveInfo, accountId: int, team
     # teamId 0 means all teams
     actual_team_id = None if teamId == 0 else teamId
 
-    team_tree = await get_team_tree(accountId, actual_team_id, sdb)
+    team_tree = await _get_team_tree(accountId, actual_team_id, sdb)
     return team_tree.to_dict()
 
 
-async def get_team_tree(
-    account: int, root_team_id: Optional[int], sdb: DatabaseLike,
-) -> TeamTree:
-    """Build the TeamTree for the Team root_team_id."""
-    team_select = [Team.id, Team.parent_id, Team.name, Team.members]
-    team_rows = await fetch_teams_recursively(
-        account, sdb, team_select, [root_team_id] if root_team_id else None,
-    )
+def build_team_tree_from_rows(team_rows: Iterable[Mapping[str, Any]],
```

Comment:
@gaetano-guerriero I extracted this function from the parent so that I can reuse it for the metrics.